ci/appveyor/win-set-cmake_generator.bat

```diff
@@ -0,0 +1,24 @@
+@echo off
+
+:set_generator_part
+if /i "%1" == "msvc15" (
+  set CMAKE_GENERATOR_PART=Visual Studio 15 2017
+  goto :set_platform_part
+)
+
+echo Invalid argument: "%1"
+exit -1
+
+:set_platform_part
+if /i "%2" == "x64" (

```

Comment:
we're not going to support other platforms
don't see a point having this as an option

Reply:
it is assertion, that fails when we choose other platforms and forget update this file